ibis/common/tests/test_graph.py

```diff
@@ -178,3 +189,75 @@ class All(Bool):
 
     copied = node.copy(arguments=(T, F))
     assert copied == All((T, F), strict=False)
+
+
+class MySequence(Sequence):
+    def __init__(self, *items):
+        self.items = items
+
+    def __getitem__(self, index):
+        return self.items[index]
```

Comment:
Are these missing test coverage?

Reply:
There are not supposed to be called, adding an assertion.

Reply:
Updated.